telemetry/src/metrics.rs

```diff
@@ -198,7 +198,7 @@ impl Metrics {
     ///
     /// # Panics
     /// Never
-    #[allow(clippy::unused_self, clippy::expect_used)]
+    #[allow(clippy::unused_self)]
     pub fn current_time(&self) -> Duration {
```

Comment:
why do we have `unused_self`?

Reply:
So that we can call this with the instance method syntax `thing.current_time`. In general this isn't a problem, it just signals that the unused parameter might be unintentional.